src/content/docs/en/tutorial/6-islands/3.mdx

```diff
@@ -65,3 +66,14 @@ Continue with our [guide to migrate this project to content collections](/en/gui
 [Join us on Discord](https://astro.build/chat)
 
 <CompletionConfetti />
+
+## Share your achievement!
+
+Congratulations on completing the Astro blog tutorial! Share your achievement with the world and let everyone know you're an 🧑‍🚀Astronaut now!
```

Comment:
I'm not 100% sure how well the positioning of the emoji will translate to screen readers. We're also already using this emoji after a sentence further up on the page, so it may be best to omit it here. Other than that, I really like this text!
```suggestion
Congratulations on completing the Astro blog tutorial! Share your achievement with the world and let everyone know you're an Astronaut now!
```

Reply:
Thanks for the review! Will edit the changes and get back 😄

Reply:
> Other than that, I really like this text!

Was really proud of that